src/Schema/Events/index.ts

```diff
@@ -915,6 +921,18 @@ export enum ActionType {
    * Corresponds to {@link SelectedItemFromSearch}
    */
   selectedItemFromSearch = "selectedItemFromSearch",
+  /**
+   * Corresponds to {@link SelectedPillFromSearch}
+   */
+  selectedPillFromSearch = "selectedPillFromSearch",
+  /**
+   * Corresponds to {@link ClickedOnRightChevronIconOnSearchPills}
+   */
+  clickedOnRightChevronIconOnSearchPills = "clickedOnRightChevronIconOnSearchPills",
+  /**
+   * Corresponds to {@link ClickedOnLeftChevronIconOnSearchPills}
+   */
+  clickedOnLeftChevronIconOnSearchPills = "clickedOnLeftChevronIconOnSearchPills",
```

Comment:
**question**: do we really want (and need) to track this action?

Reply:
I can remove it. but what do you think @abhitip ?

Reply:
Agreed, I think we can remove the chevron events!